test/src/e2e/verification.test.ts

```diff
@@ -123,14 +123,12 @@ describe("solo - 1 node", function() {
         [
             { actionType: 2, actionLength: 2 }, // Pay
             { actionType: 2, actionLength: 4 },
-            { actionType: 3, actionLength: 1 }, // SetRegularKey
+            { actionType: 3, actionLength: 1 }, // CreateShard
             { actionType: 3, actionLength: 3 },
-            { actionType: 4, actionLength: 1 }, // CreateShard
-            { actionType: 4, actionLength: 3 },
-            { actionType: 5, actionLength: 2 }, // SetShardOwners
+            { actionType: 4, actionLength: 2 }, // SetShardOwners
+            { actionType: 4, actionLength: 4 },
+            { actionType: 5, actionLength: 2 }, // SetShardUsers
             { actionType: 5, actionLength: 4 },
-            { actionType: 6, actionLength: 2 }, // SetShardUsers
-            { actionType: 6, actionLength: 4 },
             { actionType: 0x19, actionLength: 3 }, // ShardStore
```

Comment:
@somniumism You should not change these lines. These changes may cause the tests to fail.

Reply:
I fixed the cause of the test failure. However, why does this cause test failure? I changed this `verification.test.ts` because I changed `ActionTag` to new order:
original `ActionTag`
```rust
enum ActionTag {
    Pay = 0x02,
    CreateShard = 0x04,
    SetShardOwners = 0x05,
    SetShardUsers = 0x06,
    ShardStore = 0x19,
    Custom = 0xFF,
}
```
new order `ActionTag`:
enum ActionTag {
    Pay = 0x02,
    CreateShard = 0x03,
    SetShardOwners = 0x04,
    SetShardUsers = 0x05,
    ShardStore = 0x19,
    Custom = 0xFF,
}
```
So, I though I should change this `verification.test.ts`. I think it's right to modify it, but I don't understand why does this cause test failure. Could you explain why?

For the present, I modified to use the original ActionTag sequence to solve this error.